test/mitmproxy/addons/test_xss.py

```diff
@@ -0,0 +1,280 @@
+import unittest
```

Comment:
We use the fantastic pytest for testing mitmproxy, so no need for unittest here. You should get a good idea at https://github.com/mitmproxy/mitmproxy/blob/master/test/mitmproxy/test_examples.py#L129 about the style.

In a nutshell:
- Class name starts with `Test`.
- Functions start with `test_`.
- You can use plain `assert` statements. If you run with pytest (https://github.com/mitmproxy/mitmproxy#testing), that'll give you diffs if it fails, etc.